src/tic_tac_toe/core.clj

```diff
@@ -0,0 +1,7 @@
+(ns tic-tac-toe.core
+  (require [tic-tac-toe.session :refer :all]))
+
+(defn -main
+  "I don't do a whole lot ... yet."
```

Comment:
What's this???

Reply:
Changed to this:

(ns tic-tac-toe.core
  (require [tic-tac-toe.session :refer :all]))

(defn -main []
  (session-run))